docs/clients/grpc/observability.md

```diff
@@ -0,0 +1,106 @@
+# Observability
+
+The EventStoreDB gRPC clients are designed with observability in mind, offering support for OpenTelemetry. This integration provides a set of distributed traces, enabling developers to gain deeper insights into their system.
+
+::: warning
+Currently not all clients have support for observability using OpenTelemetry.
+:::
+
+::: note
+Only the append and subscribe (Catchup and Persistent) operations are instrumented.
+:::
+
+## Traces
```

Comment:
There is only one H2 in this doc. Is it necessary? Can we just introduce traces as what OpenTelemetry provides in the intro above and change the following H3s to H2s?

Reply:
In the future, our instrumentation might include [metrics](https://opentelemetry.io/docs/specs/otel/metrics/api/). If we add metrics, they will require another H2 header below 'Traces'. Therefore, I have decided to move the items that were previously below 'Traces' outside, as they apply globally.